src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -780,7 +806,12 @@ export class PostgresStorageAdapter {
     return Promise.resolve()
       .then(() => {
         fieldNames = fieldNames.reduce((list, fieldName) => {
-          const field = schema.fields[fieldName]
+          if (!{}.hasOwnProperty.call(schema.fields, fieldName)) {
+            list.push(fieldName);
+            return list;
+          }
+
+          const field = schema.fields[fieldName];
```

Comment:
Can you elaborate on this change here?

Reply:
I dont understand what you said with "elaborate on this change here".. :)

Reply:
Hmm, I'll reword it. Could you explain the reason you made the changes above here? I'm not entirely sure why you added this at first glance.

Reply:
Specifically what about the prior functionality without `...!{}.hasOwnProperty.call(schema.fields, fieldName)..` wasn't working and needed to be changed?